### PR TITLE
testing/rofi: new aport

### PR DESCRIPTION
**Pull request by xsteadfastx**

### CODE REVIEW

File: testing/rofi/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributor: Marvin Steadfast <marvin@xsteadfastx.org>
+# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
+pkgname=rofi
+pkgver=1.5.2
+pkgrel=0
+pkgdesc="A window switcher, application launcher and dmenu replacement"
+url="https://github.com/DaveDavenport/rofi"
+arch="all"
+license="MIT"
+makedepends="bash bison flex-dev pango-dev cairo-dev glib-dev librsvg-dev startup-notification-dev libxkbcommon-dev libxcb-dev xcb-util-dev xcb-util-wm-dev xcb-util-xrm-dev"
```
Comment (prspkt) [inline testing/rofi/APKBUILD]:
```suggestion
makedepends="bash bison flex pango-dev cairo-dev glib-dev i3wm-dev librsvg-dev
        startup-notification-dev libxkbcommon-dev libxcb-dev xcb-util-dev
        xcb-util-wm-dev xcb-util-xrm-dev"
checkdepends="check-dev xkeyboard-config"
```

File: testing/rofi/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+subpackages="$pkgname-dev $pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/DaveDavenport/rofi/releases/download/$pkgver/$pkgname-$pkgver.tar.gz"
```
Comment (prspkt) [inline testing/rofi/APKBUILD]:
```suggestion
source="https://github.com/davatorium/rofi/releases/download/$pkgver/rofi-$pkgver.tar.xz"
```

File: testing/rofi/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+builddir="$srcdir/$pkgname-$pkgver/"
+
+build() {
+	cd "$builddir"
+	./configure  --disable-check --prefix=/usr
+	make
+}
```
Comment (prspkt) [inline testing/rofi/APKBUILD]:
```suggestion
build() {
        cd "$builddir"
        ./configure \
                --build=$CBUILD \
                --host=$CHOST \
                --prefix=/usr \
                --sysconfdir=/etc \
                --mandir=/usr/share/man \
                --localstatedir=/var
        make
}
```

**Comment by xsteadfastx** [comment]
so did all the changes. but how do you know about all that? like the needed xkeyboard-config?

**Comment by prspkt** [comment]
the test suite was failing on one particular test regarding (key)bindings

**Pull request closed without merging**